### PR TITLE
Update libsecret to 0.20.1

### PR DESCRIPTION
**Pull request by A6GibKm**
Uses autotools to build instead of meson

### CODE REVIEW

File: ch.protonmail.protonmail-bridge.yaml [hunk 1]
```diff
@@ -7,23 +7,13 @@ separate-locales: false
 finish-args:
   - --share=ipc
   - --socket=x11
+  - --socket=wayland
```
Comment (TingPing) [inline ch.protonmail.protonmail-bridge.yaml]: Why was this added?

Comment (A6GibKm) [reply ch.protonmail.protonmail-bridge.yaml]: Tested it with `xeyes`.  Indeed, it does not use wayland, I will remove it in the next update. 

Just to clarify, is there an issue with the redundant socket enabled?

Comment (TingPing) [reply ch.protonmail.protonmail-bridge.yaml]: No harm, just odd having unused permissions.

Comment (A6GibKm) [reply ch.protonmail.protonmail-bridge.yaml]: Actually, there is harm, which I found very weird. 

I was unable to even move the window around with that flag enabled. I pushed a fix in #11.

Comment (A6GibKm) [reply ch.protonmail.protonmail-bridge.yaml]: I remember why I added that socket, protonmail-bridge.{rpm,deb} bundles wayland libraries, which is weird considering it is not using wayland.
